### ISSUE TITLE
How to calculate total-num-updates in glue ?

### ISSUE THREAD

**Issue by YuxiangLu** [pytorch/fairseq#1158]
Like RTE, it's training data size is 2490, if I train it 10 epochs with batch size 16, it will have 2490 * 10 / 16 = 1556 updates, which is different with 2036  in README.glue.md.

**Comment by ngoyal2707** [comment]
--total-num-updates are calculated in the way as you mentioned. 

In the case of `RTE`, it is slightly different due to some initial oversight in the finetuning scripts. `--total-num-updates` is only used by lr-scheduler. So in this case, the final lr at the end of training, will be slightly higher than zero. We kept the same in readme for reproducability.

Feel free to experiment with `1556` updates.
